bitnami/rabbitmq/templates/_helpers.tpl

```diff
@@ -24,6 +24,17 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 {{- end -}}
 
+{{- define "rabbitmq.labels" -}}
```

Comment:
We know that the tags we are using at https://github.com/bitnami/charts/blob/master/bitnami/rabbitmq/templates/statefulset.yaml are deprecated, but changing them will need a major version because it breaks the backwards compatibility. To be consistent with the rest of the chart, please delete this section and update the configmap with the labels we use in that file.
This is something we plan to change in the next major version.

Reply:
@miguelaeh sure, I'll do that